packages/carbon-addons-boomerang-react/src/components/BreadcrumbNav/BreadcrumbNav.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import classnames from 'classnames';
+import { settings } from 'carbon-components';
+
+const { prefix } = settings;
+
+const BreadcrumbNav = ({ className, navItems, ...rest }) => {
```

Comment:
This is a good idea but what does this provide that the carbon one doesnt? It looks like we should be able to nest RR links inside the breadcrumb component that they have?

Reply:
https://react.carbondesignsystem.com/?path=/story/breadcrumb--playground

Reply:
Is the main difference the ability to make the last item just text and not a link (in addition to being config driven)?

Reply:
Yep, being config driven and easily adding the plain text at the end would make the implementation simpler, but as you said, the composition model really makes things more clear and flexible, so I am good not to have this component at all.